### PR TITLE
ssdb: fix SHA (repack).

### PR DESCRIPTION
**Pull request by geoff-codes**
Split out from #37014 per "one commit per formula, one formula per commit" guideline.

### CODE REVIEW

**Comment by tdsmith** [comment]
Can you share a diff/get an upstream comment?

**Comment by geoff-codes** [comment]
It looks like there was an additional patch added to https://github.com/ideawu/ssdb/commits/stable-1.8.0 after the initial 1.8.0: https://github.com/Homebrew/homebrew/commits/master/Library/Formula/ssdb.rb.

Bump rev?

**Comment by tdsmith** [comment]
Good detective work. Yeah, let's bump.

**Comment by geoff-codes** [comment]
Added revision 1.

File: Library/Formula/ssdb.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
-require 'formula'
-
 class Ssdb < Formula
-  homepage "http://ssdb.io/?lang=en"
+  homepage "http://ssdb.io/"
   url "https://github.com/ideawu/ssdb/archive/1.8.0.tar.gz"
-  sha1 "ed9f016bdfef9543a866144fee4a37544f39155e"
+  sha1 "88f4930ae6f3d0234c783546e1fb00638801c4e7"
   head "https://github.com/ideawu/ssdb.git"
+  revision 1
```
Comment (mikemcquaid) [inline Library/Formula/ssdb.rb]: We don't need to revision this as it'll force everyone to reinstall it.

Comment (mikemcquaid) [inline Library/Formula/ssdb.rb]: Actually, maybe in this case we do want to.

Comment (geoff-codes) [inline Library/Formula/ssdb.rb]: Yeah [Fix ssdb-dump memory leak](https://github.com/ideawu/ssdb/commit/a59b7f2b4c9036e3d56e40ed92a782b1497bb58c) sounds like a good thing.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `ssdb: fix SHA (repacked)`.

**Pull request closed without merging**